docs/api/web-request.md

```diff
@@ -70,9 +70,10 @@ The `uploadData` is an array of `UploadData` objects.
 
 The `callback` has to be called with an `response` object.
 
-Some examples of valid `urls`:
+Some examples of valid `includeUrls`:
 
 ```js
+'<all-urls>'
```

Comment:
I think we should try to follow [existing standards](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#all_urls) here.
```suggestion
'<all_urls>'
```